v2/pkg/common/constants.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package common
+
+// =================== Config variable constants ===================
+
+const (
+	// #nosec
+	ClientSecretVar      = "AZURE_CLIENT_SECRET"
```

Comment:
I think we should make some effort to differentiate what these constants are for. Previous, when they were in their own separate packages, it was "obvious" which ones were for the Configuration secret and which were used elsewhere.

Now, it's not so obvious. It will be even less obvious if we remove `Var` suffix from them (not that I disagree with the plan to remove that suffix).

I wonder if we should either put the config specific constants into a common subpackage, like `common/config`, so that it clearly explains what goes into the configuration secret, or else we could give them a `Config` suffix (I like that less). We could also at least put them into their own file, although that won't really help clarify for users who are using them as they don't see our file structure just our package structure.

All in all, I think I tend towards `common/config`

Reply:
We could also have a `common/annotations` although I'm less certain that's a good idea.

Reply:
I'm still not used to the way Go projects sometimes deliberately have very small packages purely for naming reasons - but I think both `common/config` and `common/annotations` have merit.